src/config/keymap_action.rs

```diff
@@ -123,15 +123,74 @@ where
 #[derive(Clone, Debug, Deserialize)]
 #[serde(untagged)]
 pub enum Actions {
+    // Allows keychords to map to null, which means no actions.
+    NoAction,
```

Comment:
To my knowledge it's needed, because an enum without a value, is what `serde` regards as a `null`.

It's only used for deserialization, and converted into `KeymapAction(vec![])` right away.